### PR TITLE
Used contentUTF8 in FeignExceptionLogger

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1209

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by doncem — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/logging/FeignExceptionLogger.java [hunk 1]
```diff
@@ -13,7 +13,7 @@ public static void debugCcdException(Logger logger, FeignException exception, St
         logger.debug(
             "{}. CCD response: {}",
             introMessage,
-            exception.responseBody().map(b -> new String(b.array())).orElseGet(exception::getMessage)
+            exception.responseBody().map(b -> exception.contentUTF8()).orElseGet(exception::getMessage)
```
[the remark this answers is not in the document]
Comment (apereverzin) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/logging/FeignExceptionLogger.java:16]: `contentUTF8()` uses responseBody internally

[the remark this answers is not in the document]
Comment (doncem) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/logging/FeignExceptionLogger.java:16]: iirc, `responseBody` returns optional whereas `contentUTF8` uses `responseBody` but throws exception if empty, hence such _map or else_ approach ;)

File: src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/logging/FeignExceptionLoggerTest.java [hunk 2]
```diff
@@ -24,22 +20,19 @@
...
+            Request.create(Request.HttpMethod.POST, "url", emptyMap(), new byte[]{}, null, null),
+            "response body".getBytes()
+        );
 
         // when
         debugCcdException(logger, exception, "Intro");
 
         // then
-        verify(exception, times(1)).responseBody();
-        verify(exception, never()).getMessage();
```
Comment (rkondratowicz) [inline src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/logging/FeignExceptionLoggerTest.java:40]: test name no longer matches what's being tested

**Review by rkondratowicz — CHANGES_REQUESTED** [review]
error

**Review by rkondratowicz — APPROVED** [review]

**Comment by apereverzin** [comment]
Consistently failing in Jenkins

**Pull request closed without merging**